srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.md

```diff
@@ -599,171 +599,201 @@ frames. If only sending audio, the RECOMMENDED frequency is every
 
 This document defines an extension to DTLS-SRTP called SRTP EKT Key
 Transport which enables secure transport of EKT keying material from
-one DTLS-SRTP peer to another. This allows those peers to process EKT
-keying material in SRTP (or SRTCP) and retrieve the embedded SRTP
-keying material. This combination of protocols is valuable because it
-combines the advantages of DTLS, which has strong authentication of
-the endpoint and flexibility, along with allowing secure multiparty
-RTP with loose coordination and efficient communication of per-source
-keys.
+the DTLS-SRTP peer in the server role to the client. This allows
+those peers to process EKT keying material in SRTP (or SRTCP) and
+retrieve the embedded SRTP keying material.  This combination of
+protocols is valuable because it combines the advantages of DTLS,
+which has strong authentication of the endpoint and flexibility,
+along with allowing secure multiparty RTP with loose coordination
+and efficient communication of per-source keys.
 
 
 ## DTLS-SRTP Recap
 
-DTLS-SRTP [@!RFC5764] uses an extended DTLS exchange between two peers
-to exchange keying material, algorithms, and parameters for SRTP. The
-SRTP flow operates over the same transport as the DTLS-SRTP exchange
-(i.e., the same 5-tuple). DTLS-SRTP combines the performance and
-encryption flexibility benefits of SRTP with the flexibility and
-convenience of DTLS-integrated key and association
+DTLS-SRTP [@!RFC5764] uses an extended DTLS exchange between two
+peers to exchange keying material, algorithms, and parameters for
+SRTP. The SRTP flow operates over the same transport as the
+DTLS-SRTP exchange (i.e., the same 5-tuple). DTLS-SRTP combines the
+performance and encryption flexibility benefits of SRTP with the
+flexibility and convenience of DTLS-integrated key and association
 management. DTLS-SRTP can be viewed in two equivalent ways: as a new
-key management method for SRTP, and a new RTP-specific data format for
-DTLS.
-  
+key management method for SRTP, and a new RTP-specific data format
+for DTLS.
+
 
-## SRTP EKT Key Transport Extensions to DTLS-SRTP {#dtls-srtp-extensions}
-      
-This document defines a new TLS negotiated extension called
-"srtp\_ekt\_key\_transport" and a new TLS content type called
-EKTMessage.
+## SRTP EKT Key Transport Extensions to DTLS-SRTP
+{#dtls-srtp-extensions}
 
-Using the syntax described in DTLS [@!RFC6347], the following
-structures are used:
+This document defines a new TLS negotiated extension
+`supported_ekt_ciphers` and a new TLS handshake message type
```

Comment:
I don't like the way the quoted strings, like `ekt_key` make quoted strings in the final txt file. Can we just stay with it as is which use the ekt\_key aproach? of if we do change we need to do it all but it looks lame in the final ascii draft

Reply:
You mean just have them as plain strings?

> a new TLS negotiated extension supported_ekt_ciphers and a new...

vs.

> a new TLS negotiated extension `supported_ekt_ciphers` and a new...

I guess, but I don't really feel like changing it.  Handle in a follow-up?